target_chains/solana/programs/pyth-solana-receiver/tests/test_post_price_update_from_vaa.rs

```diff
@@ -391,7 +398,7 @@ async fn test_post_price_update_from_vaa() {
     assert_eq!(price_update_account.write_authority, poster.pubkey());
     assert_eq!(
         price_update_account.verification_level,
-        VerificationLevel::Partial { num_signatures: 13 }
+        VerificationLevel::Full
```

Comment:
this is because the transaction above errored? (maybe worth leaving a comment above this saying "check that account data isn't updated when the update fails"?)